### PR TITLE
New Port RagingBlasters

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for RagingBlasters

## Game Information
- **Title**: RagingBlasters
- **URL**: https://store.steampowered.com/app/1464550/RagingBlasters/

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — APPROVED** [review]
👍

File: ports/ragingblasters/RagingBlasters.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR=/$directory/ports/ragingblasters
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$GAMEDIR/utils/libs"
```
Comment (Cebion) [inline ports/ragingblasters/RagingBlasters.sh:28]: Wrong LD_LIBRARY_PATH, I suggest you update the template you use for your ports aswell =)

File: ports/ragingblasters/port.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "ragingblasters.zip",
+  "items": [
+    "RagingBlasters",
```
Comment (Cebion) [inline ports/ragingblasters/port.json:5]: Scriptname is wrong =)

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/ragingblasters/port.json (hunk 2 above)
Comment (Fraxinus88) [reply ports/ragingblasters/port.json]: Updated

File: ports/ragingblasters/RagingBlasters.sh (hunk 1 above)
Comment (Fraxinus88) [reply ports/ragingblasters/RagingBlasters.sh]: Updated as well... whoops

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]

**Pull request merged**